examples/tpm-sign/tpm-sign.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2014, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/go-tpm/tpm"
+)
+
+var (
+	ownerAuthEnvVar     = "TPM_OWNER_AUTH"
+	srkAuthEnvVar       = "TPM_SRK_AUTH"
+	usageAuthEnvVar     = "TPM_USAGE_AUTH"
+	migrationAuthEnvVar = "TPM_MIGRATION_AUTH"
+)
+
+func main() {
```

Comment:
This looks very similar to other example files. I'd suggest doing a git cp to make it clear how this is the same and how it differs from other examples. It makes the history cleaner that way and makes the review easier.